mgmtsystem/azure.py

```diff
@@ -297,6 +297,7 @@ def delete_stack(self, stack_name, resource_group=None):
             -DeploymentName \"{stack}\" -Force
             }}
             """.format(rg=resource_group or self.resource_group, stack=stack_name))
+        self.delete_vm(stack_name, resource_group or self.resource_group)
```

Comment:
We could have just added a vm_name as a parameter, if None, don't delete, if named, do the delete